src/tests/unit/tests/assessments/assessment-infoandexamples-title.test.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Assessments } from 'assessments/assessments';
+import { Requirement } from 'assessments/types/requirement';
+
+describe('assessment-infoandexamples-title', () => {
+    test('all infoAndExamples have a pageTitle value', () => {
+        Assessments.all().forEach(assessment => {
+            assessment.requirements.forEach(assertInfoAndExamplesHasPageTitle);
+        });
+    });
+
+    function infoAndExamplesHasPageTitle(requirement: Requirement): boolean {
+        return (
+            requirement.infoAndExamples === undefined ||
+            requirement.infoAndExamples.pageTitle !== undefined
```

Comment:
This can probably be even stronger and test equality with `requirement.name`

(maybe that suggests we should actually use a different strategy where `requirement.name` just gets passed to the panel directly, and remove the pageTitle machinery :/ )

Reply:
Made this strong as suggested & updated the instances where that equality failed.

As discussed offline, will investigate the different strategy next.